rust/main/utils/run-locally/src/main.rs

```diff
@@ -460,6 +462,21 @@ fn main() -> ExitCode {
         return report_test_result(test_passed);
     }
 
+    // Kill validator 1 and simulate a reorg, which we'll later use
+    // to ensure the relayer handles reorgs correctly.
+    stop_validator(&mut state, 1);
```

Comment:
why does the validator need to be stopped?

Reply:
ah I see, the reorg check is only done at startup time in the validator but it's ok to kill it to avoid future flakes. Will include this context in the comment